tests/resources/VCH-Util.robot

```diff
@@ -128,6 +128,26 @@ Get Docker Params
     Run Keyword If  ${port} == 2376  Set Environment Variable  VCH-PARAMS  -H ${dockerHost} --tls
     Run Keyword If  ${port} == 2375  Set Environment Variable  VCH-PARAMS  -H ${dockerHost}
 
+    # Add environment variables for Compose and TLS
+
+    ${tls_enabled}=  Get Environment Variable  DOCKER_TLS_VERIFY  ${false}
+
+    # Set environment variables if certs not used to create the VCH.  This is NOT the recommended
+    # approach to running compose.  There will be security warnings in the logs and some compose
+    # operations may not work properly because certs == false currently means we install with
+    # --no-tlsverify. Add CURL_CA_BUNDLE for a workaround in compose tests.  If we change
+    # certs == false to install with --no-tls, then we need to change this again.
+    Run Keyword If  ${tls_enabled} == ${False}  Set Environment Variable  CURL_CA_BUNDLE  ${EMPTY}
+
+    Run Keyword If  ${tls_enabled} == ${True}  Remove Environment Variable  CURL_CA_BUNDLE
```

Comment:
Can we group all the `tls_enabled == true` code together, and all the `tls_enabled == false` code in a separate block - makes it easier to see what's required in each scenario.